src/sos/__main__.py

```diff
@@ -1332,12 +1326,11 @@ def cmd_preview(args, unknown_args):
                     print(msg[1]["data"]["text/plain"])
                 elif "text/html" in msg[1]["data"]:
                     print(msg[1]["data"]["text/html"])
-                else:
-                    print("BINARY DATA of type {}".format(", ".join(
-                        msg[1]["data"].keys())))
+                if "text/plain" in msg[1]["data"]:
+                    print(msg[1]["data"]["text/plain"])
```

Comment:
This change does not match the original code.

Reply:
I reverted the changes recently.